jgrapht-core/src/main/java/org/jgrapht/graph/DirectedAcyclicGraph.java

```diff
@@ -118,20 +133,50 @@ public DirectedAcyclicGraph(
      * @param topoOrderMap the topological order map. For performance reasons, subclasses can change
      *        the way this class stores the topological order.
      * @param weighted if true the graph will be weighted, otherwise not
+     * @deprecated Deprecated
```

Comment:
Why deprecated?

Reply:
Took it back! Mostly to avoid maintaining two protected constructors.